### PR TITLE
Added table in Project Dashboard and made route for efforts

### PR DESCRIPTION
**Pull request by Sachinbisht27**
Targets #2182 

### Description
Added table for my projects in dashboard and also added user efforts in table.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/User/Entities/User.php [hunk 1]
```diff
@@ -186,4 +186,16 @@ public function activeProjects()
...
+    {
+        $userId = $this->id;
+        $efforts = [];
+        $projectTeamMembers = ProjectTeamMember::where('team_member_id', $userId)->get();
+        foreach ($projectTeamMembers as $projectTeamMember) {
+            $efforts = $projectTeamMember->current_actual_effort;
+        }
+
+        return $efforts;
+    }
```
Comment (gautam071) [inline Modules/User/Entities/User.php:200]: Can we try to incorporate with the active projects() function 
looks like there are lot of similarites

File: resources/js/components/Dashboard/UserDashboardProjects.vue [hunk 2]
```diff
@@ -1,36 +1,50 @@
...
-                        </div>
-                        <hr class="mt-1">
-                    </div>
-                </div>
-                <div v-else>
-                    <p>Fetching latest data...</p>
-                </div>
-            </div>
-        </div>
+		<div class="card ">
```
Comment (gautam071) [inline resources/js/components/Dashboard/UserDashboardProjects.vue:2]: Indentation of the file

**Review by gautam071 — CHANGES_REQUESTED** [review]
Left some comments!

File: Modules/User/Entities/User.php (hunk 1 above)
Comment (Sachinbisht27) [reply Modules/User/Entities/User.php]: Ok understand.

File: resources/js/components/Dashboard/UserDashboardProjects.vue (hunk 2 above)
Comment (Sachinbisht27) [reply resources/js/components/Dashboard/UserDashboardProjects.vue]: ok done

File: resources/js/components/Dashboard/UserDashboardProjects.vue [hunk 3]
```diff
@@ -1,45 +1,60 @@
...
+											<th scope="col2">My effort</th>
+										</tr>
+									</thead>
+									<tbody>
+										<tr>
+											<td scope="col1"><a :href="'/projects/'+project.id+'/show/'">{{ project.name }}</a></td>
+											<td scope="col2">{{efforts}}</td>
+										</tr>
+									</tbody>
+								</table>
```
Comment (gautam071) [inline resources/js/components/Dashboard/UserDashboardProjects.vue:27]: Check the indentation

**Review by gautam071 — CHANGES_REQUESTED** [review]

**Review by gautam071 — APPROVED** [review]
Looks good!

**Review by pankaj-ag — APPROVED** [review]

**Pull request closed without merging**